documentation/source/user_section/fsleyes.rst

```diff
@@ -11,6 +11,26 @@ be installed as either part of the ``FSL`` package, or as a standalone app.
 
 Previously, SCT provided instructions on how to install FSLeyes into the SCT environment. However, we now request that you install FSLeyes separately and manage the installation on your own. You can find installation instructions for FSLeyes at `this link <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLeyes>`_.
 
+.. warning::
+
+   If you choose to install FSLeyes via a complete FSL installation, you will need to update 'wxpython' (an internal packaged used by FSLeyes). This is because the version of 'wxpython' that comes with FSL is out of date (`more info here <https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3988#issuecomment-1373918661>`_).
+
+   To update, please run one of the the following commands (`depending on your FSL version <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLeyes>`_):
+
+   .. note:: The terminal commands in these instructions may require administrative privileges, depending on where you have installed FSL.
+
+   **FSL 6.0.6 or newer:**
+
+   .. code:: sh
+      $FSLDIR/bin/conda update -n base wxpython
+
+   **FSL 6.0.2 to 6.0.5.2:**
+
+   .. code:: sh
+      $FSLDIR/fslpython/bin/conda update -n fslpython -c conda-forge --update-all wxpython
```

Comment:
I have a nagging suspicion that this issue is due to FSL v6.0.6. 

- I've never encountered this issue before, even when installing FSLeyes via FSL, which perhaps suggests that in the past, FSL used to install `wxpython>4.1.1`.
- v6.0.6 introduced a rework to FSL's conda environment

If it turns out that FSL v6.0.5.2 and below are actually safe from this issue, then we could potentially shorten these instructions to only include the `6.0.6`-specific update command.

Reply:
An initial attempt to install FSL `v6.0.5.X` has failed -- it seems the `fslinstaller.py` file provided on the FSL website only supports `v6.0.6.X`:

```console
joshua@tadpole:~/Downloads$ python fslinstaller.py -d /home/joshua/fsl_6.0.5.2 --fslversion 6.0.5.2
FSL installer version: 3.2.0
Press CTRL+C at any time to cancel installation
Installation log file: /tmp/fslinstaller_0av15_8t.log

An error has occurred: FSL version "6.0.5.2" is not available - available versions: 6.0.6, 6.0.6.1, 6.0.6.2, latest
```

This is backed up by https://fsl.fmrib.ox.ac.uk/fsldownloads/fslconda/releases/.

Reply:
Ah, wait, here we are:

> To download older ( prior to FSL 6.0.6 ) versions of the FSL, the previous version of the installer can be downloaded [here](https://git.fmrib.ox.ac.uk/fsl/installer/-/raw/3.3.0/fslinstaller.py?inline=false).

Reply:
Anecdotally, the old `fslinstaller.py` is _tremendously_ slow. Right now, I'm waiting for "Stage 2" of the post-install script `fslpython_installer.sh`, which corresponds to the creation of the `fslpython` conda environment:

<details>

> ![image](https://user-images.githubusercontent.com/16181459/221269878-0ccc1d0b-b0bd-45bf-9e79-90118e67ebf6.png)

</details>

It was stuck at 0% for a good 30 minutes, and now it's stuck at 1% (also for another 30m):

<details>

> ![image](https://user-images.githubusercontent.com/16181459/221270092-07ad61ed-d004-46c1-bdd4-7cc45a1d94be.png)

</details>

Complex conda environments often take quite a long time to solve, so I wouldn't be surprised if that's what's happening here. 

Thankfully, we don't need to let this finish to determine the issue here; we can look directly at `fslpython_environment.yml` to see that `wxpython` is pinned to `4.1.1`:

![image](https://user-images.githubusercontent.com/16181459/221270672-f4616fe2-fef7-472c-bef8-cb6eed19b712.png)

This at least rules out that the issue is FSL 6.0.6-specific, meaning that we do in fact need both "update" commands.

Reply:
Ah!! Thankfully, I've realized that we don't need to let this finish to determine the issue here; we can look directly at `fslpython_environment.yml` specification, where we see that `wxpython` is pinned to `4.1.1`:

![image](https://user-images.githubusercontent.com/16181459/221270672-f4616fe2-fef7-472c-bef8-cb6eed19b712.png)

This at least rules out that the issue is FSL 6.0.6-specific, meaning that we do in fact need both "update" commands.